modules/bullet/area_bullet.h

```diff
@@ -139,6 +140,9 @@ class AreaBullet : public RigidCollisionObjectBullet {
 	_FORCE_INLINE_ void set_spOv_priority(int p_priority) { spOv_priority = p_priority; }
 	_FORCE_INLINE_ int get_spOv_priority() { return spOv_priority; }
 
+	_FORCE_INLINE_ void set_priority(int p_priority) { priority = p_priority; }
+	_FORCE_INLINE_ int get_priority() const { return priority; }
```

Comment:
These methods and the `priority` member are not needed, they are not used anywhere.